CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ should change the heading of the (upcoming) version to include a major version b
 - Updated `getDefaultFormState()` to pass true to `mergeDefaultsWithFormData` for `defaultSupercedesUndefined` when `mergeDefaultsIntoFormData` has the value `useDefaultIfFormDataUndefined`, fixing [#4322](https://github.com/rjsf-team/react-jsonschema-form/issues/4322)
 - Updated `getClosestMatchingOption()` to improve the scoring of sub-property objects that are provided over ones that aren't, fixing [#3997](https://github.com/rjsf-team/react-jsonschema-form/issues/3977) and [#4314](https://github.com/rjsf-team/react-jsonschema-form/issues/4314)
 
+## @rjsf/validator-ajv8
```

Comment:
This needs to be in a new `5.22.2` section above

Reply:
Thanks for the quick review, just updated the changelog as suggested :)